Jellyfin.Api/Auth/FirstTimeSetupPolicy/FirstTimeSetupHandler.cs

```diff
@@ -36,37 +27,14 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             if (!_configurationManager.CommonConfiguration.IsStartupWizardCompleted)
             {
                 context.Succeed(requirement);
-                return Task.CompletedTask;
             }
-
-            var contextUser = context.User;
-            if (requirement.RequireAdmin && !contextUser.IsInRole(UserRoles.Administrator))
+            else if (requirement.RequireAdmin && !context.User.IsInRole(UserRoles.Administrator))
             {
                 context.Fail();
-                return Task.CompletedTask;
             }
-
-            var userId = contextUser.GetUserId();
-            if (userId.IsEmpty())
-            {
-                context.Fail();
-                return Task.CompletedTask;
-            }
-
-            if (!requirement.ValidateParentalSchedule)
-            {
-                context.Succeed(requirement);
-                return Task.CompletedTask;
-            }
-
-            var user = _userManager.GetUserById(userId);
-            if (user is null)
-            {
-                throw new ResourceNotFoundException();
-            }
-
-            if (user.IsParentalScheduleAllowed())
+            else
             {
+                // Any user-specific checks are handled in the DefaultAuthorizationHandler.
```

Comment:
Are those triggered when this one succeeds?

Reply:
The DefaultAuthorizationHandler is called before the FirstTimeSetupHandler

Even if the FirstTimeSetupHandler calls Succeed on the requirement the call will still result in a 403